### PR TITLE
Updated XIRRUS-MIB; added Temperature sensor

### PR DESCRIPTION
**Pull request by k-y**
Updated XIRRUS-MIB to latest version from Riverbed
Added Temperature sensor into includes/definitions/discovery/xirrus_aos.yaml

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/definitions/discovery/xirrus_aos.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+mib: XIRRUS-MIB
+modules:
+  sensors:
+    temperature:
+      data:
+        -
+          oid: systemTempCelsius
+          num_iod: .1.3.6.1.4.1.21013.1.2.28.21.1.3.
```
Comment (laf) [inline includes/definitions/discovery/xirrus_aos.yaml]: This should be `num_oid`

**Comment by k-y** [comment]
Hmm, how do I get the CLA signed for my e-mail?  It's showing signed for my github user.

**Comment by laf** [comment]
You'll need to add the email address from the first commit to your github account (how to is linked in the cla response above).

Thanks for adding the test data, could you maybe add more by running the commands in the link I added so that we have more test data for this device.

**Comment by k-y** [comment]
Yup, I added my email to my account but the CLA is still not updating.

**Comment by k-y** [comment]
@murrant that did the trick, thanks

**Pull request merged**